src/pages/docs/test-management/settings/smtp-configuration.md

```
@@ -0,0 +1,72 @@
+
+**Send Testsigma notifications from your trusted email domain by setting up SMTP for improved deliverability & control.**
+
+---
+
+Configuring Simple Mail Transfer Protocol (SMTP) allows your organization to send email notifications from your own authenticated email address instead of Testsigma’s default sender. This ensures improved branding, deliverability, and security for automated notifications. This article discusses configuring SMTP in Test Management by Testsigma.
+
+---
+
+> ## **Prerequisites**
+> Before you begin, ensure that you have the following details from your email service provider:
+>    1. SMTP server address
```

markdownlint-cli2 (0.18.1)

Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)